src/hearings/containers/request-hearing/hearing-timing/hearing-timing.component.ts

```diff
@@ -114,6 +114,11 @@ export class HearingTimingComponent extends RequestHearingPageFlow implements On
     };
   }
 
+  public getHearingWindow(): HearingWindowModel {
```

Comment:
This function is duplicated. Can you move it into some service that is imported by both hearing requirements component and hearing timing component? For example you could put it in ValidatorUtils. Sonar will probably complain if you don't